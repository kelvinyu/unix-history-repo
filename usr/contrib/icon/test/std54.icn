record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("every write(repl(\"a\" | \"ab\" | \"ba\",1 to 5)) ----> ",image(every write(repl("a" | "ab" | "ba",1 to 5))) | "none")
   write("repl(\"\",0) ----> ",image(repl("",0)) | "none")
   write("repl(&cset,0) ----> ",image(repl(&cset,0)) | "none")
   write("repl(\"abc\",-1) ----> ",image(repl("abc",-1)) | "none")
   write("trim(&lcase) ----> ",image(trim(&lcase)) | "none")
   write("trim(&lcase,&lcase) ----> ",image(trim(&lcase,&lcase)) | "none")
   write("trim(3.14159,58) ----> ",image(trim(3.14159,58)) | "none")
   write("trim(&lcase,[]) ----> ",image(trim(&lcase,[])) | "none")
end
