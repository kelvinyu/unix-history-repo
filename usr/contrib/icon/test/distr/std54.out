a
aa
aaa
aaaa
aaaaa
ab
abab
ababab
abababab
ababababab
ba
baba
bababa
babababa
bababababa
every write(repl("a" | "ab" | "ba",1 to 5)) ----> none
repl("",0) ----> ""
repl(&cset,0) ----> ""
Run-time error 205 at line 12 in std54.icn
value out of range
offending value: -1
